src/app/pages/apptrust/apptrust.page.ts

```diff
@@ -13,20 +10,49 @@ declare var navigator: any;
     styleUrls: ['./apptrust.page.scss'],
 })
 
-export class AppTrustPage implements OnInit {
-  private static readonly METRICS_KEY = 'security';
+export class AppTrustPage {
+  public serverUrl: string[];
 
-  ngOnInit() {
-  }
+  constructor(private securityService: SecurityService, private alertController: AlertController) {
 
-  public isAvailable() {
-      // return this.platform.is('cordova');
-      return true;
   }
 
+  public clientInit() {
+    this.securityService.clientInit()
+    .then(clientData => {
+      if (clientData.data.disabled) {
+        this.alertController.create({
+          header: 'Application Trust',
+          subHeader: 'Mobile Security Service',
+          message: clientData.data.disabledMessage,
+          buttons: ['OK']
+        }).then(alert => {
+          alert.present();
+        });
+      } else {
+        this.alertController.create({
+        header: 'Application Trust',
+          subHeader: 'Mobile Security Service',
+          message: 'This application is currently enabled',
+          buttons: ['OK']
+        }).then(alert => {
+          alert.present();
+        });
+      }
+    })
+    .catch(Error => {
```

Comment:
```suggestion
    .catch(() => {
```

Reply:
As I am not doing anything with Error, :+1:

Reply:
going to console log the error as it is not always a internet connection issue and would be nice to have.